src/interfaces/tasks-i.ts

```diff
@@ -11,7 +12,7 @@ export interface TaskApiHeaderI {
   comments: any[]
   commentsCount: number
   commentsLink: string
-  date: string // submitted date
+  date: ApiDateTimeUtc // submitted date
```

Comment:
Just a clarification. (It's a string anyway.)

<img width="610" height="165" alt="image" src="https://github.com/user-attachments/assets/91cbca86-51aa-43dd-b7a6-0060b4412d5b" />